test/skate_web/controllers/page_controller_test.exs

```diff
@@ -79,5 +80,17 @@ defmodule SkateWeb.PageControllerTest do
 
       assert html_response(conn, 200) =~ "div id=\"app\""
     end
+
+    @tag :authenticated
+    setup do
+      reassign_env(:skate, :record_sentry, true)
+      reassign_env(:skate, :sentry_environment, "test_env")
+    end
```

Comment:
Note that the contents of the `setup` block are run before every `test` inside of a given `describe` - see the docs [here](https://hexdocs.pm/ex_unit/1.13/ExUnit.Callbacks.html#setup/1). I'm not actually sure exactly how that behaves when the `setup` block is placed partway through the test cases instead of at the top, but in any event since this setup is only relevant for the one Sentry-related test you should probably move those two calls to `reassign_env` into the test case itself and get rid of the `setup` block.

Reply:
done!